layers/core_checks/cc_image.cpp

```diff
@@ -2193,24 +2187,22 @@ bool CoreChecks::ValidateGetImageSubresourceLayout(VkDevice device, const IMAGE_
 
             if (!is_valid) {
                 const char *vuid =
-                    is_ext ? "VUID-vkGetImageSubresourceLayout2EXT-tiling-02271" : "VUID-vkGetImageSubresourceLayout-tiling-02271";
-                skip |= LogError(image_state.image(), vuid,
-                                 "%s: VkImageSubresource.aspectMask (%s) must be "
-                                 "VK_IMAGE_ASPECT_MEMORY_PLANE_i_BIT_EXT, with i being less than the "
-                                 "VkDrmFormatModifierPropertiesEXT::drmFormatModifierPlaneCount (%" PRIu32
-                                 ") associated with the image's format (%s) and "
-                                 "VkImageDrmFormatModifierPropertiesEXT::drmFormatModifier (%" PRIu64 ").",
-                                 caller, string_VkImageAspectFlags(aspect_mask).c_str(), max_plane_count,
+                    is_2 ? "VUID-vkGetImageSubresourceLayout2KHR-tiling-02271" : "VUID-vkGetImageSubresourceLayout-tiling-02271";
+                skip |= LogError(vuid, image_state.image(), loc.dot(Field::aspectMask),
+                                 "is %s and drmFormatModifierPlaneCount (%" PRIu32
+                                 ") which associated with the image's format (%s) and "
+                                 "drmFormatModifier (%" PRIu64 ").",
```

Comment:
Wording looks weird to me, I suggest something along those lines:
```suggestion
                skip |= LogError(vuid, image_state.image(), loc.dot(Field::aspectMask),
                                 "is %s, hence the plane index is either unspecified or superior or equal to the drmFormatModifierPlaneCount (%" PRIu32
                                 ") associated with the image's format (%s) and "
                                 "drmFormatModifier (%" PRIu64 ").",
```

Reply:
agree, re-worded it